File: test/tw/rmstudio/mt/java/course/mcu/ds/ex1/test/TestSelectionSort.java
package tw.rmstudio.mt.java.course.mcu.ds.ex1.test;

import tw.rmstudio.mt.java.course.mcu.ds.ex1.sorting.SelectionSort;

public class TestSelectionSort extends TestSortingAlgorithm {
    @Override
    int[] sort(int[] randomNumbers) {
        return new SelectionSort().sortResult(randomNumbers).getSortedNumbers();
    }
}

File: test/tw/rmstudio/mt/java/course/mcu/ds/ex1/test/TestSelectionSort.kt
package tw.rmstudio.mt.java.course.mcu.ds.ex1.test

import tw.rmstudio.mt.java.course.mcu.ds.ex1.sorting.SelectionSort

class TestSelectionSort : TestSortingAlgorithm() {
    override fun sort(randomNumbers: IntArray) = SelectionSort().sortResult(randomNumbers).sortedNumbers
}
